app/languages/languages.js

```diff
@@ -9,14 +10,22 @@ import { getLanguages } from 'react-native-i18n';
 // Step 5 - Add the labels to be used in respective json files. The labels are the key and the content is the value in different language, so make sure for each file the key remains the same
 // Step 6 - In React Native code import the main languages file and call the translate function - languages.t('label.labelname')
 
+import delabels from './de';
 import enlabels from './en';
+import eslabels from './es';
+import frlabels from './fr';
+import ptBRlabels from './pt-BR';
 
 // This will fetch the user's language
 let userLang = undefined;
-getLanguages().then(languages => {
-  userLang = languages[0].split('-')[0]; // ['en-US' will become 'en']
-  i18next.changeLanguage(userLang);
+const lang = RNLocalize.getLocales();
+lang.map((ln) => {
+  userLang = ln.languageCode;
```

Comment:
This results into `userLang = 'en' `on my German devices - maybe because `RNLocalize.getLocales()` returns the following in my iOS simulator:

```
[{"countryCode": "US", "isRTL": false, "languageCode": "de", "languageTag": "de-US"}, 
{"countryCode": "US", "isRTL": false, "languageCode": "fr", "languageTag": "fr-US"}, 
{"countryCode": "BR", "isRTL": false, "languageCode": "pt", "languageTag": "pt-BR"}, 
{"countryCode": "US", "isRTL": false, "languageCode": "es", "languageTag": "es-US"}, 
{"countryCode": "US", "isRTL": false, "languageCode": "it", "languageTag": "it-US"}, 
{"countryCode": "US", "isRTL": false, "languageCode": "en", "languageTag": "en-US"}]
```

Reply:
Maybe we have to use `react-native-i18n` as in the Treecounter app?

Reply:
If a correct language tag is not available, I think we can have it fallback to first two letters. eg: from de-US --> `de`

Reply:
Do you think `lang[0].map((ln) => { userLang = ln.languageCode}` will solve these issue?

Reply:
@norbertschuler react-native-i18n is deprecated.

Reply:
Currently its falling back to `en`

Reply:
Ok, I did a small adaption. It works now also on my iOS simulator with several images.